sources/web/datalab/polymer/components/datalab-app/datalab-app.ts

```diff
@@ -55,6 +55,14 @@ class DatalabAppElement extends Polymer.Element {
 
     // Will be called after the custom element is done rendering.
     window.addEventListener('WebComponentsReady', this._boundResizeHandler, true);
+
+    ApiManager.disconnectedHandler = () => {
```

Comment:
Hmmm.. my thinking was that the notification will show for success events, which necessarily require the connection to be established. Do you have a scenario in mind where we would need to show a notification for something that happens client-only?